exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -132,6 +132,8 @@
   OptionValidator PARQUET_VECTOR_FILL_CHECK_THRESHOLD_VALIDATOR = new PositiveLongValidator(PARQUET_VECTOR_FILL_CHECK_THRESHOLD, 100l, 10l);
   String PARQUET_NEW_RECORD_READER = "store.parquet.use_new_reader";
   OptionValidator PARQUET_RECORD_READER_IMPLEMENTATION_VALIDATOR = new BooleanValidator(PARQUET_NEW_RECORD_READER, false);
+  String PARQUET_READER_INT96_AS_TIMESTAMP = "store.parquet.int96_as_timestamp";
```

Comment:
Should rename this to store.parquet.reader.abc to make it clear this is a reader only property